components/reddit/sources/new-comments-on-a-subreddit/new-comments-on-a-subreddit.js

```diff
@@ -0,0 +1,107 @@
+const common = require("../../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-on-a-subreddit",
+  name: "New comments on a subreddit",
+  description:
+    "Emits an event each time a new comment is added to a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    subreddit: {
+      propDefinition: [reddit, "subreddit"],
+    },
+    subreddit_id36: {
+      type: "string",
+      label: "ID36 of Subreddit",
+      description:
+        "Pipedream will emit a new event when a comment is posted to the subreddit pointed by this ID36.",
+      optional: false,
+    },
+    context: {
+      type: "integer",
+      label: "Context (0 to 8)",
+      description:
+        "If set to 0, it will include the new comment in the emitted event. Otherwise, it specifies how many parents of the new comment will be included in said event.",
```

Comment:
I'd rephrase to something like this:
```js
module.exports = {
  props: {
    context: {
      // Other attributes here...
      description: `
        When set to 0, the emitted events will only contain the new comment.
        Otherwise, the events will also contain the parents of the new comment
        up to the number indicated in this property.
      `,
    },
  },
};
```

Reply:
Right, I updated the comment. Slightly modified in one instance where min value is 2.